packages/moonstone/Scrollable/Scrollable.js

```diff
@@ -454,25 +454,23 @@ class ScrollableBase extends Component {
 			if (isPageUp(keyCode) || isPageDown(keyCode)) {
 				Spotlight.setPointerMode(false);
 				direction = this.getPageDirection(keyCode);
-				overscrollEffectRequired = !this.scrollByPage(direction);
-			} else {
-				direction = getDirection(keyCode);
-				if (direction) {
-					const element = Spotlight.getCurrent();
+				overscrollEffectRequired = this.scrollByPage(direction);
+			} else if (getDirection(keyCode)) {
+				const element = Spotlight.getCurrent();
 
-					overscrollEffectRequired = !(element ? getTargetByDirectionFromElement(direction, element) : null);
-					if (overscrollEffectRequired) {
-						const {horizontalScrollbarRef, verticalScrollbarRef} = this.uiRef;
+				direction = getDirection(keyCode);
+				overscrollEffectRequired = !(element ? getTargetByDirectionFromElement(direction, element) : null);
+				if (overscrollEffectRequired) {
```

Comment:
Should we need this?

Reply:
This condition is required for some edge cases. For example, an effect should not show up when pressing 5-way _UP_ button after a scroller touched the top edge by pressing _OK_ button on paging control _up_ button. Without this conditional block, an effect shows up. Thanks for pointing out.